index.html

```diff
@@ -887,6 +887,14 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
           resource, then the server MUST supply a default <a>ACL</a>. The default <a>ACL</a> resource SHOULD be located
           in the same server as the controlled resource.
         </p>
+        <blockquote class="informative">
+          <p>
+            Non-normative note:
+            The predicate <code>acl:default</code> in the [[WEBAC]] vocabulary indicates authorizations that should be
+            inherited. The [[SOLIDWEBAC]] documentation is being updated to use this in place of the obsolete predicate
```

Comment:
wouldn't it be better to reference the version of the SOLID spec that "is being updated" to avoid potential confusion after that spec is updated?

Reply:
I'm hoping that this issue will be resolved (and the comment removed) before we finalize the Fedora spec... but I agree that adding the version number might avoid potential confusion.

Reply:
@zimeon that new text is great, thanks so much!